reference-controllers/rabbitmq-controller/src/main.rs

```diff
@@ -163,138 +140,88 @@ async fn reconcile_server_config_map(
         .map_err(Error::ReconcileConfigMapFailed)?;
 
     if server_config_o.is_some() {
-        info!(
-            "Current rv of server configmap: {}",
-            server_config_o
-                .as_ref()
-                .unwrap()
-                .metadata
-                .resource_version
-                .as_ref()
-                .unwrap()
-        );
+        let resource_version = server_config_o
+            .as_ref()
+            .unwrap()
+            .metadata
+            .resource_version
+            .as_ref()
+            .unwrap()
+            .clone();
+        info!("Current rv of server configmap: {}", resource_version);
         if server_config.data != server_config_o.as_ref().unwrap().data {
```

Comment:
I would say we can just go ahead to update the configmap without checking whether the data is the same or not -- if the data remains the same then the update won't change anything.